src/Lexer.php

```diff
@@ -396,7 +410,7 @@ protected function scanEOS()
     /**
      * Returns keywords for provided type.
      *
-     * @param string $type Keyword type
+     * @phpstan-param TKeywordsType $type Keyword type
```

Comment:
This method also becomes deprecated and unused in #350
```suggestion
     * @param string $type Keyword type
```